Marlin/src/lcd/menu/menu_advanced.cpp

```diff
@@ -230,7 +236,12 @@ void menu_backlash();
       #endif // EXTRUDERS > 1
     #endif
 
+    #ifdef FILAMENT_RUNOUT_DISTANCE_MM
+      MENU_MULTIPLIER_ITEM_EDIT(float3, "Runout Distance mm", &lcd_runout_distance_mm, 1,  30);
+    #endif
+
     END_MENU();
+    runout.set_runout_distance(lcd_runout_distance_mm);
```

Comment:
This should not be done here, but instead should be done in an edit callback. Search for `MENU_ITEM_EDIT_CALLBACK` to see several examples of how to do this.

Reply:
OK...